Source/ACE.DatLoader.Tests/DatTests.cs

```diff
@@ -24,5 +25,63 @@ public void LoadPortalDat_NoExceptions()
             int count = dat.AllFiles.Count();
             Assert.AreEqual(79694, count);
         }
+
+        [TestMethod]
```

Comment:
These take an hour+ to run and leave 800,000 files on your computer. Please comment out the TestMethod attribute at a minimum - i want to be able to run tests safely.